serverless/security/serverless-kafka-security.adoc

```diff
@@ -0,0 +1,18 @@
+[id="serverless-kafka-security"]
+= Security configuration for Knative Kafka
+include::modules/common-attributes.adoc[]
+include::modules/serverless-document-attributes.adoc[]
+:context: serverless-kafka-security
+
+toc::[]
+
+In production, Kafka clusters are often secured using the TLS or SASL authentication methods. This section shows how to configure a Kafka channel to work against a protected Red Hat AMQ Streams cluster using TLS or SASL.
```

Comment:
```suggestion
In production, Kafka clusters are often secured using the SSL/TLS or SASL authentication methods. This section shows how to configure a Kafka channel to work against a protected Red Hat AMQ Streams cluster using SSL/TLS or SASL.
```
See [SSG](https://www.ibm.com/docs/en/ibm-style?topic=word-usage#f) rules for SSL/TSL

Reply:
I don't see where this is listed in the style guide? Also, I think adding this would make the docs inaccurate, and TLS is used both in IBM docs and our docs all the time. E.g.:

https://www.ibm.com/docs/en/zos/2.5.0?topic=protocols-tls-ssl
https://docs.openshift.com/container-platform/4.9/virt/virtual_machines/importing_vms/virt-tls-certificates-for-dv-imports.html

Not sure where this guidance comes from, but I think it's incorrect.